README.rst

```diff
@@ -255,9 +255,114 @@ Since going over all (or select) keys in an S3 bucket is a very common operation
   >>> prefix = 'annual/monthly_rain/'
   >>> for key, content in s3_iter_bucket(bucket, prefix=prefix, accept_key=lambda key: '/201' in key, workers=1, key_limit=3):
   ...     print(key, round(len(content) / 2**20))
-  annual/monthly_rain/2010.monthly_rain.nc 14
-  annual/monthly_rain/2011.monthly_rain.nc 14
-  annual/monthly_rain/2012.monthly_rain.nc 14
+  annual/monthly_rain/2010.monthly_rain.nc 13
+  annual/monthly_rain/2011.monthly_rain.nc 13
+  annual/monthly_rain/2012.monthly_rain.nc 13
+
+
+Migrating to the New `open` Function
```

Comment:
@mpenkov what happened here? I remember commenting on this line in a review, and you fixing it. But now it's back in the README?